src/http/server.ts

```diff
@@ -385,7 +425,9 @@ export class Server {
    *     Returns the response as stringified JSON. This is only used for unit
    *     testing purposes.
    */
-  public handleHttpRequestForFavicon(request: any): string {
+  public handleHttpRequestForFavicon(
+    request: Drash.Http.Request,
+  ): Drash.Interfaces.ResponseOutput {
```

Comment:
Has the doc block been updated to reflect this?

Reply:
updated